p2p/discovery.py

```diff
@@ -93,7 +100,13 @@ def __init__(self,
     async def lookup_random(self, cancel_token: CancelToken) -> List[kademlia.Node]:
         return await self.kademlia.lookup_random(self.cancel_token.chain(cancel_token))
 
-    def get_random_nodes(self, count: int) -> Generator[kademlia.Node, None, None]:
+    def get_random_bootnode(self) -> Generator[kademlia.Node, None, None]:
```

Comment:
`def get_random_bootnode(self) -> Iterable[kademlia.Node]` seems like a better fit for the signature